wildfire/lib/screens/all_updates/all_updates.dart

```diff
@@ -0,0 +1,352 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:wildfire/screens/dashboard/blankpage.dart';
+import 'package:wildfire/screens/dashboard/dashcontainers.dart';
+// import 'package:wildfire/model/report_model.dart';
+
+class AllUpdates extends StatefulWidget {
+  const AllUpdates({Key? key}) : super(key: key);
+
+  @override
+  _AllUpdatesState createState() => _AllUpdatesState();
+}
+
+class _AllUpdatesState extends State<AllUpdates> {
+  int _currentIndex = 1;
+  final List<Widget> index = [
```

Comment:
You don't need this here, it is of no use. This is for the bottom menu implementation which is done only ones